src/index.js

```diff
@@ -13,6 +13,6 @@ export { default as SplashScreen } from './containers/SplashScreen';
 export { default as Pagination } from './containers/Pagination';
 export { default as ControlledSwitch } from './containers/ControlledSwitch';
 export { default as UncontrolledSwitch } from './containers/UncontrolledSwitch';
-export { UsernameTranslation } from './containers/UsernameTranslation'
+export { Credentials } from './containers/UsernameTranslation'
```

Comment:
I haven't changed the containers name yet, as the latest changes would not be visible since it would be considered a new file, but it will be changed to `export { Credentials } from './containers/Credentials'` before potential merging

Reply:
I think it would not rewrite the whole file to the point that it would be seen as a new one and the changes would not be visible. Git is clever enough to avoid this.